structured_external_storage_all/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Structured External Storage All Attachments
+===========================================
+
+Create ir.attachment.metadata records for the attachments to sync to respective
+clouds: Dropbox, Google Drive, Amazon S3 Bucket.
+
+Installation
+============
+- Just install
+
+Configuration
+=============
+
+- Create the Sync Rules in *Settings->External Backup->Attachment Sync Rules*
```

Comment:
And before that, add any models you want to sync attachments for in "Referencable Models".

Also add a small text on how to sync historic records.